tolstoy/src/schema.rs

```diff
@@ -80,6 +96,14 @@ pub mod tests {
             },
             (_, _) => { panic!("Wrong number of results."); },
         }
+
+        let partitions = SyncMetadataClient::get_partitions(&tx).unwrap();
```

Comment:
In general, we can't be `:db.part/user`-centric, 'cuz `:db.part/tx` advances too.  I think this will get simpler as you generalize, not more complicated.